ui.py

```diff
@@ -1,11 +1,13 @@
 from PyQt5.QtGui import QFont
 from PyQt5.QtWidgets import *
 from PyQt5.QtCore import pyqtSlot
+from pygame import mixer
+from speech_recognizer import speech_recognizer as sr
```

Comment:
the same names confuse where the class implementation and where the imported library